tools/tune/crossover/crossover_gen_coefs.m

```diff
@@ -29,13 +29,13 @@
 
 % Generate the a,b coefficients for a second order
 % low pass butterworth filter
-function lp = lp_iir(fs, fc, gain_db)
+function lp = lp_iir(fs, fc, ~)
```

Comment:
Please explain why gain_db is removed? It does not look like Matlab compatibility change.

Reply:
First option: Input argument 'gain_db' is unused, so this is OK, to be replaced with '~',  There is a typographical error in the declaration i.e. the body of the function . The body of the function does not use specified input argument.
or
Second option:  `lp_iir()` should be 2 input argument instead of 3. but this need change across the all files across the  code base. In the current code, the calling routines are filled with '0' all over.

So I opted first option